docs/tutorial.rst

```diff
@@ -59,24 +61,28 @@ Your shell prompt should change once you are in the virtual env to show (cci) at
 
 If you would like to automatically activate the virtual environment whenever you open a new terminal, you can add the activate command to your ``~/.bash_profile``.
 
-For more information about virtual environments in Python, see the Python Packaging User Guide: https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments
-
 Windows
 ^^^^^^^
 
+Ensure you are in your home directory (C:\Users\your.username) and execute the following commands:
+
 .. code-block:: powershell
 
-    python -m venv ~\cci
-    ~\cci\Scripts\Activate.ps1
+    python -m venv cci
+    cci\Scripts\activate
```

Comment:
@robertbwatson were you using PowerShell or not? We may need to provide different instructions based on which shell the user is working in. (But, it's also entirely possible that the PowerShell commands were incorrect, since I didn't have access to test this when I wrote it.)

Reply:
No, just the command prompt. Maybe that's why it wasn't working. Salesforce employees on Windows are prohibited from running PowerShell.

Reply:
You want to be careful with this as well, as the command will create the folder in whatever directory one happens to be in. For command prompt, I believe you want to use `python -m venv %userprofile%/cci` . I am going to confirm that in short order. That should place cci in something approaching the 'correct' location, rather than just buried in a random folder.

Reply:
This is outdated. The new commit I made references Git Bash instead of the command prompt. When you open a new Git Bash I believe you are automatically placed in your home directory by default, but I could be wrong.

That being said, I opened this PR because I wanted to make some quick changes to make it less confusing for the Windows setup but it sounds like there might be grander changes coming, so I don't know if anything will actually happen to this PR.